README.md

```diff
@@ -3,7 +3,7 @@ Starter middleware, extensions, and tools for building APIs when working with Or
 
 ## Installation
 ```
-npm i @ordercloud/catalyst
+npm install @ordercloud/catalyst
```

Comment:
`npm i` is totally valid, but may not be common knowledge. Changing it to install makes it more descriptive